website/content/docs/concepts/seal.mdx

```diff
@@ -8,145 +8,130 @@ description: >-
-that would be provided with Shamir. The process remains the same.
+It is still possible to seal a Vault node using the API. When you use the API, Vault remains sealed until you restart it or use the unseal API. Auto
+unseal requires the recovery key fragments instead of the unseal key fragments
+that the Shamir seal provides. The process remains the same.
 
-For a list of examples and supported providers, please see the
+For a list of examples and supported providers, refer to the
 [seal documentation](/vault/docs/configuration/seal).
 
-When DR replication is enabled in Vault Enterprise, [Performance Standby](/vault/docs/enterprise/performance-standby) nodes on the DR cluster will seal themselves, so they must be restarted to be unsealed.
+When you enable DR replication in Vault Enterprise, [performance standby](/vault/docs/enterprise/performance-standby) nodes on the DR cluster seal themselves, so you must restart them to unseal them.
 
 <Warning title="Recovery keys cannot decrypt the root key">
 
-Recovery keys cannot decrypt the root key and thus are not sufficient to unseal
-Vault if the auto unseal mechanism isn't working. They are purely an authorization mechanism.
-Using auto unseal creates a strict Vault lifecycle dependency on the underlying seal mechanism. 
-This means that if the seal mechanism (such as the Cloud KMS key) becomes unavailable, 
-or deleted before the seal is migrated, then there is no ability to recover 
-access to the Vault cluster until the mechanism is available again. **If the seal 
+Recovery keys cannot decrypt the root key and therefore are not sufficient to unseal
+Vault if the auto unseal mechanism isn't working.
+Using auto unseal creates a strict Vault lifecycle dependency on the underlying seal mechanism.
+If a seal mechanism such as the Cloud KMS key becomes unavailable
+or is deleted before you migrate the seal, you cannot recover
+access to the Vault cluster until the mechanism is available again.
+
+**If the seal
 mechanism or its keys are permanently deleted, then the Vault cluster cannot be recovered, even
 from backups.**
-To mitigate this risk, we recommend careful controls around management of the seal
+To mitigate this risk, we recommend careful controls around managing the seal
 mechanism, for example using
 [AWS Service Control Policies](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html)
 or similar.
-With Vault Enterprise secondary clusters (disaster or performance) can have a
-seal configured independently of the primary, and when properly configured guards
-against *some* of this risk.  Unreplicated items such as local mounts could still
-be lost.
+
+With Vault Enterprise, you can configure a secondary seal for disaster or performance clusters independently of the primary seal. When you properly configure a secondary seal, it guards
+against some of this risk.  You could still lose unreplicated items such as local mounts.
```

Comment:
Passive voice, removes bold for emphasis. We can keep the bold for emphasis if it is necessary, but I do not think it is here.